packages/apps/composer-app/package.json

```diff
@@ -24,15 +24,20 @@
     "@dxos/sentry": "workspace:*",
     "@dxos/text-model": "workspace:*",
     "@sentry/react": "^7.17.2",
+    "@types/showdown": "^2.0.0",
     "phosphor-react": "^1.4.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-router-dom": "^6.4.0"
+    "react-drag-drop-files": "^2.3.8",
+    "react-router-dom": "^6.4.0",
+    "showdown": "^2.1.0",
+    "turndown": "^7.1.1"
```

Comment:
`showdown` is capable of both import and export, however when exporting it appears to need non-trivial intervention in order to parse our styled HTML correctly, whereas `turndown` converts with only one trivial issue which I've resolved in this PR.

Reply:
if you think it'd be better to use showdown for export long term perhaps leave a todo noting what the issue is

Reply:
Done 👍